src/collection-browser.ts

```diff
@@ -647,6 +647,7 @@ export class CollectionBrowser
         label=${this.manageViewLabel}
         showSelectAll
         showUnselectAll
+        ?enableRemoveButton=${this.dataSource.checkedTileModels.length !== 0}
```

Comment:
suggestion: rename to `removeAllowed` so that we can get closer to the responsibility.

`enableRemoveButton` => `removeAllowed`